backend/crypt/crypt.go

```diff
@@ -700,6 +701,50 @@ func (f *Fs) ChangeNotify(ctx context.Context, notifyFunc func(string, fs.EntryT
 	do(ctx, wrappedNotifyFunc, pollIntervalChan)
 }
 
+var commandHelp = []fs.CommandHelp{
+	{
+		Name:  "encode",
+		Short: "Encode the given filename(s)",
```

Comment:
This could probably do with an example to be maximally user friendly as could "decode"